en/release-notes/upgrade-guide-v2.1.md

```diff
@@ -0,0 +1,164 @@
+
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_D5EB83DE4EB14ED7F4E0412694594BEA563C6F613AB6CAC04191666891BDD83C_1543326015234_image.png)
+
+
+**Mesh Renderer Component**
+Mesh Renderer is used to draw grid resources. See [Mesh Renderer](https://docs.cocos2d-x.org/creator/2.1/manual/en/3d/mesh-renderer.html).
+
+![](https://github.com/cocos-creator/creator-docs/raw/v2.1/zh/3d/img/mesh_renderer.png)
+
+**Bone Animation Component**
+The **Skeletal Animation** component inherits from the **Animation** component. The **Skeletal Animation** component is not much different from the **Animation** component. The exception is the **Skeletal Animation** component uses a clip that can only be a **Skeletal Animation** clip. The **Skeletal Animation** component is automatically added to the model's Prefab when it is imported. See [Bone Animation Component](https://docs.cocos2d-x.org/creator/2.1/manual/en/3d/skeleton-animation.html) for details.
+
+![](https://github.com/cocos-creator/creator-docs/raw/v2.1/en/3d/img/search-skeleton-animation.png)
+
+For more 3D instructions, please consult the documentation [3D System](https://docs.cocos2d-x.org/creator/2.1/manual/en/3d/index.html).
+
+**Optimize automatic gallery build speed**
+__v2.1__ greatly optimizes the speed of auto-atlas construction. If the atlas has not changed, the results of the last build are used.
+
+**Further improve rendering performance**
+Between __v2.0__ compared to __v1.x__, there are some cases where the engine does not render batches, resulting in an increase in draw calls. __v2.1__ re-optimized these conditions. If the previous **color** or **opacity** cannot be batch processed, **sliced** and **non-sliced** share the same texture and cannot be batched. The problem that the **opacity** 0 node still occupies the draw call is solved.
+
+**Add simulator debugging function**
+In the **Project Settings** -> **Preview Run Panel**, the **Open simulator debugger panel** option and the **Wait for debugger connect** option have been added. The purpose of the **Open simulator debugger panel** option is to automatically open the debug window when the simulator previews the project. The **Wait for debugger connect** option is used to suspend the boot process until the debugger connection is complete for debugging the load process.
```

Comment:
1. It is best to have a blank line between the title and the body
2. **Project Settings -> Preview Run Panel** -> **Settings -> Preview Run** panel

Reply:
2. thank you.